.github/workflows/deploy.yml

```diff
@@ -0,0 +1,31 @@
+name: deploy
+
+on: [published]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+      - name: configure python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.7'
+      - name: configure node
+        uses: actions/setup-node@v1
+        with:
+            node-version: '10.x'
+      - name: install
+        run: |
+          pip install -e .[all]
+          npm install -g eslint
+          sudo apt-get install -y graphviz
+      - name: build
+        run: |
+          make html
+      - name: deploy
+        uses: peaceiris/actions-gh-pages@v3
+          with:
```

Comment:
Tested on my clone after modifying the file a bit and sending a pull request to my own clone (though accidentally created a pull request to this repository too).

@oliver-sanders it complained about this `with` because of its indentation - https://github.com/kinow/cylc-sphinx-extensions/actions/runs/57041154

![image](https://user-images.githubusercontent.com/304786/76801928-7a27a880-683b-11ea-8b63-ffbdc71d7e4c.png)

If you fix it, then rebase and push to this branch, I think the action should work as the PR #15 was merged.

Reply:
Nice test, fixed indentation...